src/triggers.js

```diff
@@ -151,6 +164,18 @@ export function getRequestObject(triggerType, auth, parseObject, originalParseOb
     request.original = originalParseObject;
   }
 
+  if (triggerType === Types.beforeSave || triggerType === Types.afterSave) {
+    // Adds ability to get set the context, which makes a copy of the object
```

Comment:
comment is a little confusing.  `Adds ability to get and set the context. Both will clone the object.`